users/serializers.py

```diff
@@ -0,0 +1,33 @@
+from rest_framework import serializers
+from  django.contrib.auth import get_user_model
+
+
+UserModel = get_user_model()
+
+
+class UserSerializer(serializers.ModelSerializer):
+    password = serializers.CharField(min_length=8, max_length=50, style={'input_type': 'password'}, write_only=True)
+    password2 = serializers.CharField(min_length=8, max_length=50, style={'input_type': 'password'}, write_only=True)
+
+    class Meta:
+        model = UserModel
+        fields = ('username', 'password', 'password2', 'email')
```

Comment:
Some questions to think about:
1) Is the "email" field unique? 
2) Which of the fields (email or username) would be used for login? Do we need both on signup?
3) How do we check if the user is the real owner of the email address?